### PR TITLE
feat: handle parsing of compound crs

### PR DESCRIPTION
**Pull request by roelofvandijkO**

### CODE REVIEW

File: s2p/rpc_utils.py [hunk 1]
```diff
@@ -102,11 +103,20 @@ def min_max_heights_from_bbx(im, lon_m, lon_M, lat_m, lat_M, rpc, exogenous_dem_
...
+    if proj_crs.is_compound:
+        assert len(proj_crs.sub_crs_list) == 2, f"CRS too complex to parse {proj_crs.sub_crs_list}"
+        assert len([x for x in proj_crs.sub_crs_list if x.is_vertical]) == 1, f"Found more than 1 vertical CRS {proj_crs.sub_crs_list}"
+        horizontal_crs = [x for x in proj_crs.sub_crs_list if not x.is_vertical][0]
+        vertical_datum = [x for x in proj_crs.sub_crs_list if x.is_vertical][0]
+        crs_epsg = f"epsg:{horizontal_crs}+{vertical_datum}"
+    else:
+        crs_epsg = proj_crs.to_epsg()
     # convert lon/lat to im projection
     x_im_proj, y_im_proj = geographiclib.pyproj_transform([lon_m, lon_M],
```
Comment (nsfabina) [inline s2p/rpc_utils.py:116]: This line and the next few makes me wonder whether pre-commit is installed and running

**Review by nsfabina — APPROVED** [review]
Thank you very much for handling this

File: s2p/rpc_utils.py (hunk 1 above)
Comment (roelofvandijkO) [reply s2p/rpc_utils.py:116]: Hopefully not

Comment (nsfabina) [reply s2p/rpc_utils.py:116]: Oh yes, wrong repo, thank you

**Pull request merged**